packages/chevrotain/src/parse/grammar/checks.ts

```diff
@@ -56,45 +57,40 @@ import dropRight from "lodash/dropRight"
 import compact from "lodash/compact"
 import { tokenStructuredMatcher } from "../../scan/tokens"
 
+export function validateLookahead(options: {
+  lookaheadStrategy: ILookaheadStrategy
+  rules: Rule[]
+  maxLookahead: number
+  tokenTypes: TokenType[]
+  grammarName: string
+}): IParserDefinitionError[] {
+  if (typeof options.lookaheadStrategy.validate !== "function") {
```

Comment:
I'm not 100% sure, but a PEG style lookahead strategy should be able to get by without any specific validations. However, it would probably be better to have mandatory `validate` method, even if it's a NOOP.

Reply:
Yeah I prefer an end user to consciously and **explicitly** decide to have no `lookahead` validations.